integration_test/docker/test-docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+
+services:
+  core0:
+    container_name: core0
+    image: "ghcr.io/celestiaorg/celestia-app-ephemeral:latest"
+    entrypoint: [
+      "/bin/bash"
+    ]
+    command: [
+      "/start-celestia-appd.sh"
+    ]
+    volumes:
+      - ${PWD}/docker/start-celestia-appd.sh:/start-celestia-appd.sh:ro
+      - ${PWD}/docker/fix-genesis.sh:/fix-genesis.sh:ro
+      - keyring-volume:/celestia-app/node/keyring-test
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.10
+
+  bridge0:
+    container_name: bridge0
+    depends_on:
+      - core0
+    image: "ghcr.io/celestiaorg/celestia-node:sha-25652db"
```

Comment:
Shouldn't this point to latest releases? and also in Celestia-node add an image tagged latest?
this would always test against the same version If I understand right

Reply:
makes sense :+1: